# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/grepmail/hotfix-install.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- grepmail-5.3111/Makefile.PL.vanilla	2025-06-24 16:15:00.328285387 +0200
+++ grepmail-5.3111/Makefile.PL	2025-06-24 16:15:03.459084924 +0200
@@ -29,8 +29,6 @@
 # Perl 5.6 doesn't work with URI::Escape. We get an error about Exporter not exporting "import"
 perl_version '5.008';
 
-check_custom_installation();
-
 install_script('grepmail');
 
 configure_requires(
